os/linux-based.md

```diff
@@ -68,7 +70,7 @@ qemu-system-x86_64 --enable-kvm -m 4096 -cdrom <legacy>.iso
 This will start the legacy image in QEMU with KVM (Kernel-based Virtual Machine) enabled, with 4096 MiB of RAM, booting the legacy ISO. If you are on a Windows host, remove the `--enable-kvm` flag, as it does not apply to the Windows platform.
 ## The System - In-Depth
 ![dahliaOS Linux-Based 201215 Software architecture chart, Showing the overall layout of the system](assets/charts/arch.png)
-Although Linux and Zircon could not be more different, dahliaOS Linux blends the security, efficiency, and virtualization capabilities of Zircon with a lightweight Linux base system.
+Although Linux and Zircon could not be more different, dahliaOS Linux blends the security, efficiency, and virtualization capabilities of Zircon with a lightweight Linux based system.
```

Comment:
The "base system" is Linux, so this line does not need to be changed.

Reply:
"lightweight linux base system" seems very odd, but if you want, I'll change it back.

Reply:
It puts "Zircon" on a  "Linux base system", therefore the _result_ is a "Linux based system".
I'll do it, thank you for the contribution :smiley:

Reply:
yeah its not like the "linux-based" system, its the "base" or underlying system, idk tho based might ring better

Reply:
Please check https://github.com/dahlia-os/documentation/pull/78